### DISCUSSION TITLE
How to take screenshots of multiple screens

### DISCUSSION THREAD

**Discussion by ulisses-cruz** [flameshot-org/flameshot#3286] [Q&A]
I like this application. But recently I got a multi screen setup but am able to take screenshots of only one screen.
Is there away that I can configure it to take screenshots of other screens?

**Comment by mmahmoudian** [comment]
Yes, if you are using Flameshot in `gui` mode, you can use <kbd>Ctrl</kbd><kbd>a</kbd> to "select all" everything visible. Alternatively you can use `flameshot full` to capture the full visible area.

Of course without knowing what is your operating system, we cannot help you more, but if you are using Linux or macOS, this page can be useful:

https://flameshot.org/docs/advanced/commandline-options/

**Comment by madrats** [comment]
If you are using Windows, then currently you have to restart Flameshot **every time** your screen configuration changes. There are multiple open issues about it as well, hopefully next version will include a fix.
